ble-events-handler.js

```diff
@@ -0,0 +1,315 @@
+/**
+ * The `CONFIG` object contains a scenes property which is an array of scene objects.
+ * Each scene object consists of two properties: `conditions` and `action`. The `conditions` property
+ * defines the conditions under which the scene should be triggered, and the `action`
+ * property defines the function to be executed when the conditions are met.
+ *
+ * The `conditions` are defined as key-value pairs.
+ * These keys correspond to specific data values received with the event.
+ * The values associated with these keys can be either a direct value, an object specifying a comparison, or a function that
+ * must return boolean value.
+ * The `conditions` value supports various types: 
+ * - key: value pair, where the key is fetched from the received data and must equal to the target value
+ * - function that receives the current value and must return boolean
+ * - object with `compare` and `value` keys:
+ * Where `compare` supports the following methods: 
+ * - "==" -> both values are the same
+ * - "<" -> the current value is less than the target value
+ * - ">" -> the current value is bigger than the target value
+ * - "~=" -> the rounded value of both values are the same
+ * - "!=" -> both values are different types
+ * - "in" -> (supplied value must be array) the current value is IN the array
+ * - "notin" -> (supplied value must be array) the current value is NOT IN the array
+ *
+ * The `action` property defines a function that receives event's data as an input. You can write custom code within this function to
+ * perform specific actions.
+ *
+ * The following example will show how to handle events from Shelly BLU devices what are emited from `ble-shelly-blu.js` example.
+ * This script CAN'T be used standone for BLE events.
+ *
+ * Event name for the `ble-shelly-blu.js` example is `shelly-blu`.
+ */
+
+/****************** START CHANGE ******************/
+let CONFIG = {
+  // List of scenes
+  scenes: [
+    /** SCENE START 1 - Shelly BLU example **/
+    {
+      // when event name is `shelly-blu` and button is pressed more than 0 times
+      conditions: {
+        event: "shelly-blu",
+        mac: "XX.XX.XX.XX.XX.XX",
```

Comment:
use something like 12:34:56:78:90, add comment that the user should change it
Is the mac formatting emitted with a dot separator or with column?

Reply:
Fixed https://github.com/ALLTERCO/shelly-script-examples/pull/45/commits/f5ea82db896f0bff1e53cacb1f47ed43fb78b49b